Sourcery/Parsing/Composer.swift

```diff
@@ -112,8 +173,8 @@ struct Composer {
                 tupleElement.type = resolveType(typeName: tupleElement.typeName, containingType: containingType, unique: unique, modules: modules, typealiases: typealiases)
             }
         }
-        return unique[lookupName.unwrappedTypeName]
-            ?? typeFromModule(lookupName.unwrappedTypeName, modules: modules)
+
+        return unique[lookupName.unwrappedTypeName] ?? unique[lookupName.generic?.name ?? ""] ?? typeFromModule(lookupName.unwrappedTypeName, modules: modules)
```

Comment:
because type from different module can be referenced with module name, which is not part of `unique` keys. `typeFromModule` solves that.